### DISCUSSION TITLE
📢 🪲 Rectangle Pro unable to create application group via Save Current Layout when CleanShot X is running

### DISCUSSION THREAD

**Discussion by omair-inam** [rxhanson/RectanglePro-Community#80] [Q&A]
For any [CleanShot X](https://cleanshot.com/) users out there, I've discovered that I cannot create an application group using **Save Current Layout** when CleanShot X is running.  

* I've logged #79 and uploaded [a video](https://share.getcloudapp.com/p9uOYz9E) demonstrating the issue.
* @rxhanson any immediate thoughts about what might be happening? 

Interestingly, Rectangle Pro seems to work fine with CloudApp running in the background (which is what I used to record the video).. However, CleanShot X has better UX, so it would be amazing if the two could work harmoniously together 😛

**Comment by rxhanson** [comment]
Sorry for the delayed response here, and thanks for the details. I’m getting ready to work through a Rectangle Pro sprint and will see what I can do. Unfortunately this could be one of those scenarios where a good solution isn’t possible, but I do agree that it would be nice to have Rectangle Pro work unimpeded.
